src/configuration-requestor.ts

```diff
@@ -17,7 +17,9 @@ export default class ConfigurationRequestor {
     }
 
     await this.flagConfigurationStore.setEntries(configResponse.flags);
-    if (Object.keys(configResponse.bandits ?? {}).length) {
+    const flagsHaveBandits = Object.keys(configResponse.bandits ?? {}).length > 0;
+    const banditStoreProvided = !!this.banditConfigurationStore;
+    if (flagsHaveBandits && banditStoreProvided) {
```

Comment:
This is the change I snuck in; It will let us do things like let us have our Client JS SDK not fetch bandits even if there are bandit flags.